tools/add_colab_instruction.py

```diff
@@ -9,7 +9,15 @@
     "Since Java is not natively supported by Colab, we need to run the following code to enable Java kernel on Colab.\n",
     "\n",
     "1. Run the cell bellow (click it and press Shift+Enter),\n",
-    "2. then refresh the page (press F5) right after that.\n",
+    "2. If you want to use the GPU with MXNet, refresh the page (press F5) and stay at Python runtime on GPU. Run the following commands:\n"
```

Comment:
Can you add some qualifiers about the DJL version here? Explain why this is being done. Otherwise, we may update DJL and then forget to update this document and users will be confused after reading it

Reply:
Good point! I made some changes. Let me know if I need to add further details.